### DISCUSSION TITLE
Website is mojibaked

### DISCUSSION THREAD

**Discussion by mqudsi** [adamghill/unsuckjs.com#41] [General]
Hello,

The main website at unsuckjs.com has become mojibaked. It is correctly served as utf-8 correctly but it seems there was an issue in how it was statically built; the text is utf-8 that has been incorrectly saved as `latin-1`/`windows-1252`/`iso-8859-1`.

The issue appears under Chrome and Firefox, and you can confirm it's a mojibake issue by trying to decode the document as `iso-8859-1` instead of `utf-8` and observe that non-ascii text remains broken (but in a different way). 

It's not immediately obvious; only when you look at the names of projects that include non-ansi/ascii characters, e.g.

![image](https://github.com/adamghill/unsuckjs.com/assets/606923/2160cf50-25f1-4f8d-9e2b-589d6ec36414)

and

![image](https://github.com/adamghill/unsuckjs.com/assets/606923/2c41c76e-2a97-4b76-b664-f5cee7b615d4)

Verification of mojibake charsets:

![image](https://github.com/adamghill/unsuckjs.com/assets/606923/a291db2f-c92a-4e38-967b-989fe007b45c)

**Comment by adamghill** [comment]
Interesting, do you know if this happened with a recent change? I don't remember seeing this previously.

**Reply by mqudsi** [reply]
It was recent; I want to say it was fine as of c3e57b25bf24c0e399c8508cdb166b150bf0ba3e.

**Comment by adamghill** [comment]
This was because I switched to a JSON5 library in a recent version of `coltrane` and it wasn't parsing with UTF-8 by default. https://github.com/adamghill/coltrane/releases/tag/0.32.1 should fix this and I am updating the dependencies for unsuckjs now and re-deploying the site.

It does appear to be fixed with that change, thanks for letting me know!

<img width="204" alt="image" src="https://github.com/adamghill/unsuckjs.com/assets/317045/f418ba7a-d91b-4d4b-8729-687e47fc18ef">
